cmd/inspect/malware.go

```diff
@@ -80,6 +81,19 @@ func executeMalwareAnalysis() error {
 
 	packageVersion := purl.PackageVersion()
 
+	// If package version is empty or latest replace it with actual literal latest version
+	// Reference: https://github.com/safedep/vet/issues/446
+	if packageVersion.GetVersion() == "" || packageVersion.GetVersion() == "latest" {
```

Comment:
Can you add an E2E test for this. Have a look at:

https://github.com/safedep/vet/blob/main/test/scenarios/all.sh
https://github.com/safedep/vet/blob/main/test/scenarios/scenario-9-malware-analysis.sh